.npmignore

```diff
@@ -1,6 +1,8 @@
+.*
+appveyor.yml
+spec
 spec-cordova
 spec-plugman
```

Comment:
I believe those two are ancient history and can be removed. No blocker though.

Reply:
I wasn't going risk it but I couldn't find anything on it, nor have I seen it. I think I will got ahead and remove it.

Reply:
```
spec-cordova
spec-plugman
```

has been dropped.